README.md

```diff
@@ -20,7 +20,7 @@ on.
 The IoT Ops CLI is part of the Azure CLI extension index. Installation is as simple as:
 
 ```
-az extension add --name azure-iot-ops
+az extension add --upgrade --name azure-iot-ops
```

Comment:
does this work with the b versions?

Reply:
It should
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/28658112/1ba62f20-b10e-436c-a925-9f976f3af197)